commands/fun/trivia.js

```diff
@@ -0,0 +1,67 @@
+const Social = require(`${process.cwd()}/base/Social.js`);
+const Discord = require("discord.js");
+const snek = require("snekfetch");
+const B64 = require("base-64");
+class Trivia extends Social {
+  constructor(client) {
+    super(client, {
+      name: "trivia",
+      description: "Delivers a quiz with the correct answer awarding 10 points.",
+      category: "Fun",
+      usage: "trivia",
+      extended: "Ready to wrack your brain? ",
+      cooldown: 10,
+      aliases: ["quiz"],
+      botPerms: ["EMBED_LINKS"]
+    });
+  }
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+
+    const inf = await snek.get("https://opentdb.com/api.php?amount=10&category=9&difficulty=medium&type=multiple&encode=base64");
+    const ob = await inf.body;
```

Comment:
`inf.body` is a getter, but it does not return a Promise. Therefore, the await is next to useless.